### PR TITLE
[M1020] include duplicate attr/growth_form with validation result

### PR DESCRIPTION
**Pull request by gridcell**

### CODE REVIEW

**Comment by gridcell** [comment]
[bleaching duplicate highlighting](https://trello.com/c/AI4CGTq8/1020-bleaching-duplicate-highlighting)

File: src/api/submission/validations.py [hunk 1]
```diff
@@ -945,18 +947,27 @@ def validate_percent_values(self):
             percent_soft = ob.get("percent_soft") or 0
             percent_algae = ob.get("percent_algae") or 0
             values = [percent_hard, percent_soft, percent_algae]
-            if self._validate_percent_values(values, "validate_percent_values") == ERROR:
+            if (
```
Comment (saanobhaai) [inline src/api/submission/validations.py]: Is using parentheses in these spots really more readable/pythonic?

File: src/api/submission/validations.py [hunk 2]
```diff
@@ -989,16 +1000,30 @@ def validate_colony_count(self):
         return self.ok(self.identifier)
 
     def validate_duplicate_genus_growth(self):
-        # ERROR
+         # ERROR
```
Comment (saanobhaai) [inline src/api/submission/validations.py]: extra space

File: src/api/submission/validations.py [hunk 3]
```diff
@@ -989,16 +1000,30 @@ def validate_colony_count(self):
...
         obs = self.get_observations(self.data) or []
+
+        dup_obs = [ob for n, ob in enumerate(obs) if ob in obs[:n]]
+        dup_obs_genus_growth = [
+            dict(
+                attribute=dup_ob.get("attribute"), growth_form=dup_ob.get("growth_form")
```
Comment (saanobhaai) [inline src/api/submission/validations.py]: Do we need to protect against `None`s here?

File: src/api/submission/validations.py (hunk 1 above)
Comment (gridcell) [reply src/api/submission/validations.py]: Black formatting

**Pull request merged**